generator.properties

```diff
@@ -0,0 +1,8 @@
+#mysql.connector=C:/Users/cheke/.m2/repository/mysql/mysql-connector-java/5.1.47/mysql-connector-java-5.1.47.jar
+mysql.connector=C:/Users/Tianfang Ma/.m2/repository/mysql/mysql-connector-java/8.0.15/mysql-connector-java-8.0.15.jar
+mysql.url=jdbc:mysql://127.0.0.1:3306/upmc?serverTimezone=America/New_York&useSSl=false
+mysql.user=root	
+mysql.password=matian@VRF30805
```

Comment:
Please change password right now

Reply:
This file is redundant after the merge #83 . I have deleted it. The original properties files are under "/resources" not root.